external/storm-elasticsearch/README.md

```diff
@@ -84,6 +84,7 @@ Provided components (Bolt, State) takes in EsConfig as a constructor arg.
 |---	|--- |---
 |clusterName | Elasticsearch cluster name | String (required) |
 |nodes | Elasticsearch nodes in a String array, each element should follow {host}:{port} pattern | String array (required) |
+|additionalParameters | Additional Elasticsearch configuration parameters | Map<String, String> (optional) |
```

Comment:
@alexpanov It would be better to describe that it will be applied to TransportClient.